docs/team/nseah21.md

```diff
@@ -1,87 +1,63 @@
 ---
 layout: page
-title: Project Portfolio Page for Nicholas Seah
+title: Nicholas Seah's Project Portfolio Page
 ---
 
-## Overview
+### Project: RC4HDB
 
-**RC4 Housing Database** offers a convenient and intuitive interface for RC4 housing management staff to streamline their daily operations.
-## Summary of Contributions
+RC4HDB is a housing database that offers a convenient and intuitive interface for housing management staff to streamline their daily operations. 
 
-### Code contributed:
+Given below are my contributions to the project.
 
-I contributed code to the following modules/classes:
-1. *Coming soon*
+* **New Feature**: Added the ability to show/hide table columns. 
+    * What it does: allows the user to hide any number of unwanted columns in the table. Columns that have been hidden can be restored by using the reset command.
+    * Justification: This feature improves the product significantly because a user can de-clutter their screen and only focus on the information that they require, saving them time and effort in their work.
```

Comment:
This is up to your own discretion, but I would not really recommend putting the justification here. Maybe you could forward them to the portion in the DG? or some other place. This is just my own opinion, it's also fine to keep it here.